### DISCUSSION TITLE
Shall we remove bestzip?

### DISCUSSION THREAD

**Discussion by vicary** [floydspace/serverless-esbuild#223] [Q&A] [answered]
See https://github.com/serverless-heaven/serverless-webpack/pull/911, the core will try compare code changes and only deploy the delta. See especially this review comment https://github.com/serverless-heaven/serverless-webpack/pull/911#pullrequestreview-717773822.

Is it a good idea to follow suit and revert https://github.com/floydspace/serverless-esbuild/issues/96?

**Comment by olup** [comment, marked as answer]
Right now it's not used by default, only when you pass on an option. It solves the original poster of #96 issue with performances, but it seems most people won't need it or be willing to pay the price (especially in terms of zip change detection).

**Reply by olup** [reply]
See #187

**Reply by vicary** [reply]
Thanks for the explanation!
